tidb-resource-control.md

```diff
@@ -0,0 +1,138 @@
+
+### Prerequisites
+
+To create, modify, or delete a resource group, you need to have the `SUPER` or `RESOURCE_GROUP_ADMIN` privilege.
+
+You can create a resource group in the cluster with [`CREATE RESOURCE GROUP`](/sql-statements/sql-statement-create-resource-group.md), and then bind the users to a specific resource group by using [`CREATE USER`](/sql-statements/ sql-statement-create-user.md) or [`ALTER USER`](/sql-statements/sql-statement-alter-user.md).
+
+### Step 1. Enable the resource control feature
+
+Enable flow control for the resource group.
+
+```sql
+SET GLOBAL tidb_enable_resource_control = 'ON';
+```
+
+In TiKV, set the parameter `resource_control.enabled` to `true`. The parameter `resource_control.enabled` is disabled by default. You need to contact the [PingCAP support team](/tidb-cloud/tidb-cloud-support.md) to enable it.
+
+### Step 2. Create a resource group, and bind users to it
+
+Resource group quotas are expressed as [RU (Resource Unit)](/tidb-RU.md), which is TiDB's unified abstraction of CPU, IO, and other system resources.
+
+The following is an example of how to create a resource group and bind users to it.
+
+1. Create a resource group `rg1` with a quota of 500 RU per second for read requests and 300 RU per second for write requests, and allow applications in this resource group to use excessive resources when system resources are available.
+
+    ```sql
+    CREATE RESOURCE GROUP IF NOT EXISTS rg1
+    RRU_PER_SEC = 500
+    WRU_PER_SEC = 300
+    BURSTABLE
+    ;
+    ```
+
+2. Create a resource group `rg2` with a quota of 600 RU per second for read requests and 400 RU per second for write requests, and do not allow applications in this resource group to use excessive resources even when system resources are available.
+
+    ```sql
+    CREATE RESOURCE GROUP IF NOT EXISTS rg2
+    RRU_PER_SEC = 600
+    WRU_PER_SEC = 400
+    ;
+    ```
+
+3. Bind users `usr1` and `usr2` to resource groups `rg1` and `rg2` respectively.
+
+    ```sql
+    ALTER USER usr1 RESOURCE GROUP rg1;
+    ```
+
+    ```sql
+    ALTER USER usr2 RESOURCE GROUP rg2;
+    ```
+
+After you complete the preceeding operations, the resource consumption by newly created sessions is controlled by the specified quota. Read requests are limited by the quota of the read RU, and write requests are limited by the quota of the write RU. If the system load is relatively high and there is no spare capacity, the resource consumption rate of both users will be strictly controlled not to exceed the quota. Meanwhile, the consumption ratio of RU metrics for both users' read and write requests is basically proportional to the specified quota. When system resources are abundant, the resource consumption rate of `usr1` is allowed to exceed the quota because it has set `BURSTABLE`.
```

Comment:
```suggestion
After you complete the preceeding operations, the resource consumption by newly created sessions is controlled by the specified quota. Read requests are limited by the read RU quota, and write requests are limited by the write RU quota. If the system load is relatively high and there is no spare capacity, the resource consumption rate of both users will be strictly controlled not to exceed the quota. Meanwhile, the consumption ratio of RU metrics for both users' read and write requests is basically proportional to the specified quota. When system resources are abundant, the resource consumption rate of `usr1` is allowed to exceed the quota because it has set `BURSTABLE`, while `usr2` is not allowed.
```